commit/plugin_functions.go

```diff
@@ -35,17 +35,18 @@ func observeLatestCommittedSeqNums(
 	sort.Slice(knownSourceChains, func(i, j int) bool { return knownSourceChains[i] < knownSourceChains[j] })
 	latestCommittedSeqNumsObservation := make([]plugintypes.SeqNumChain, 0)
 	if readableChains.Contains(destChain) {
-		lggr.Debugw("reading latest committed sequence from destination")
-		onChainLatestCommittedSeqNums, err := ccipReader.NextSeqNum(ctx, knownSourceChains)
+		lggr.Debugw("reading sequence numbers", "chains", knownSourceChains)
+		onChainNextSeqNums, err := ccipReader.NextSeqNum(ctx, knownSourceChains)
```

Comment:
It took me a good 5 mins to understand this line. Its not clear at all that this makes a call to the destination chain, I think we should make our names more clear

Reply:
With the multiramp contracts the naming has kind of been lost but I prefer making this `ExpectedNextSequenceNumbers` or if we want to shorten it `ExpectedNextSeqNums`.

Another name I'd be fine with, because it lines up with the contracts, is `OfframpMinSeqNrs` which makes it clear we are reading from the dest + makes it clear what it is reading (min seq nr being defined as the expected next seq nr for the future commit report).

Reply:
there's a todo to include destChainSelector in the interface. that way it will become cleaner, where it reads from